### PR TITLE
Uplift k8s to 1.22.0

### PR DESCRIPTION
**Pull request by kashifest**
Uplift k8s to 1.22.0
Also uplift crictl to 1.21.2

### CODE REVIEW

**Review by Rozzii — DISMISSED** [review]

File: ci/scripts/image_scripts/install_crio_on_ubuntu.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 set -uex
 
-export CRICTL_VERSION=${CRICTL_VERSION:-"v1.21.0"}
+export CRICTL_VERSION=${CRICTL_VERSION:-"v1.21.2"}
```
Comment (fmuyassarov) [inline ci/scripts/image_scripts/install_crio_on_ubuntu.sh:5]: same here `1.22.0`

**Review by fmuyassarov — CHANGES_REQUESTED** [review]

File: ci/scripts/image_scripts/install_crio_on_centos.sh [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 set -uex
 
-export CRICTL_VERSION=${CRICTL_VERSION:-"v1.21.0"}
+export CRICTL_VERSION=${CRICTL_VERSION:-"v1.21.2"}
```
[the remark this answers is not in the document]
Comment (kashifest) [reply ci/scripts/image_scripts/install_crio_on_centos.sh]: Done

File: ci/scripts/image_scripts/install_crio_on_ubuntu.sh (hunk 1 above)
Comment (kashifest) [reply ci/scripts/image_scripts/install_crio_on_ubuntu.sh]: Done

Comment (fmuyassarov) [reply ci/scripts/image_scripts/install_crio_on_ubuntu.sh]: Thank you @kashifest  :)

**Review by fmuyassarov — APPROVED** [review]

**Pull request merged**